loader/dd_library_loader.c

```diff
@@ -256,13 +261,15 @@ static void ddloader_telemetryf(telemetry_reason reason, const char *format, ...
         return;  // parent
     }
 
-    char *points = "";
+    char points_buf[256] = {0};
+    char *points = points_buf;
     switch (reason) {
         case REASON_ERROR:
-            points =
-                "\
-                {\"name\": \"library_entrypoint.error\"}, \"tags\": [\"error_type:NA\"]}\
-            ";
+            snprintf(points_buf, sizeof(points_buf), "\
+                    {\"name\": \"library_entrypoint.error\", \"tags\": [\"error_type:%s\"]}\
+                ",
+                error ? error : "NA"
```

Comment:
In case `error` points to a char[] which is "too long", this would leave you with an invalid JSON in `points_buf`. I would like to somehow make this more explicit, I guess a comment for `ddloader_telemetryf` would already be okay.